bicycleparameters/cycle_app.py

```diff
@@ -88,17 +93,17 @@
                       dbc.Row([dbc.Col(dcc.Loading(id='geometry-load',
                                                    type='dot',
                                                    children=[html.Div(id='geometry-bin',
-                                                                      children=[html.Img(src='',
+                                                                      children=[html.Div([dcc.Graph(                    #html.Img(src='',
                                                                                          id='geometry-plot',
-                                                                                         className='img-fluid')])]),
+                                                                                         className='img-fluid')])])]),    #Change html.Img to Dcc.graph
                                        lg=5,
                                        width=12),
                                dbc.Col(dcc.Loading(id='eigen-load',
                                                    type='dot',
                                                    children=[html.Div(id='eigen-bin',
-                                                                      children=[html.Img(src='',
+                                                                      children=[html.Div([dcc.Graph(  
                                                                                          id='eigen-plot',
-                                                                                         className='img-fluid')])]),
+                                                                                         className='img-fluid')])])]), #Changed html.Img (matlipplot)  to Dcc.graph (plotly)
```

Comment:
In general, there is no need to have these comments in the final code. They helped you but won't mean anything to anyone else, because we are swapping plotly graph for the image method.